src/main/java/com/revature/backend/util/StagingListener.java

```diff
@@ -1,15 +1,18 @@
-package com.revature.backend.AssignmentManagerUtil;
+package com.revature.backend.util;
```

Comment:
@RevatureGentry `AssignmentManagerUtil` was renamed to `util`, but Git is preventing a merge.  I'm unsure how to resolve this